interfaces/tempo_cluster/v1/schema.py

```diff
@@ -0,0 +1,94 @@
+"""This file defines the schemas for the provider and requirer sides of this relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+from enum import Enum
+from typing import Optional, Dict, List
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel, Field, Json
+
+
+class TempoClusterProviderAppData(BaseModel):
+    """TempoClusterProviderAppData."""
+    worker_config: Json[str] = Field(
+        description="The tempo configuration that the requirer should run with."
+                    "Yaml-encoded. Must conform to the schema that the presently deployed "
+                    "workload version supports; for example see: "
```

Comment:
Yeah valid point.
The way we thought about it:
since Tempo (coordinator and worker) is meant to be deployed as one solution, the coordinator generates the config, then gets validated with pydantic models, and sends it as a raw config over this interface `tempo_cluster` for the dumb worker to run it as is. If the schema changed, the worker would fail to start -> set blocked status.
In case we know there's a major schema version update, we can have some logic in the worker to update the config sent over `tempo_cluster` sth like
```
if ConfigModelV1.validate(config_from_reldata):
    unit.status = ActiveStatus("Please upgrade the tempo coordinator to v2 (rev > 1412)")
   conf = ConfigModelV2.from_v1(config_from_reldata) 
else:
    conf = ConfigModelV2(config_from_reldata)
```